### PR TITLE
Fixing Bugs Post Canonical Book Page Launch

### PR DESCRIPTION
**Pull request by tabshaikh**
Closes: #2652 #2241 
Updates: #3561 
-  Edit Form Revamped
  -- Tabs in excerpts and links removed and added in work details
  -- work cover added 
  -- Order shuffled in edition information
-  Work subtitle added
-  Edition info duplicated in description
-  Edit button added to edition info

fix

### Screenshot

bug_fix

### CODE REVIEW

File: openlibrary/plugins/openlibrary/js/readmore.js [hunk 1]
```diff
@@ -18,4 +18,39 @@ export function initReadMoreButton() {
             $(this).addClass('restricted-height');
         }
     });
+
```
Comment (cclauss) [inline openlibrary/plugins/openlibrary/js/readmore.js:21]:
```suggestion

```

File: openlibrary/plugins/openlibrary/js/readmore.js [hunk 2]
```diff
@@ -18,4 +18,39 @@ export function initReadMoreButton() {
...
+    //         });
+    //         $('.work-menu.work-details').addClass('selected');
+    //     }
+    //     else if(isElementInViewport(document.getElementsByClassName('edition-info'))) {
+    //         console.log('In editions info');
+    //         $('.work-menu li').each(function(){
+    //             $(this).removeClass('selected');
+    //         });
+    //         $('.work-menu.edition-details').addClass('selected');
+    //     }
```
Comment (cclauss) [inline openlibrary/plugins/openlibrary/js/readmore.js:44]:
```suggestion
    //     }
```

File: openlibrary/plugins/openlibrary/js/readmore.js [hunk 3]
```diff
@@ -18,4 +18,39 @@ export function initReadMoreButton() {
...
+    //             $(this).removeClass('selected');
+    //         });
+    //         $('.work-menu.edition-details').addClass('selected');
+    //     }   
+    // });
+
+    // function isElementInViewport(el) {
+    //     console.log(el);
+    //     var rect = el.getBoundingClientRect();
+
```
Comment (cclauss) [inline openlibrary/plugins/openlibrary/js/readmore.js:50]:
```suggestion

```

File: openlibrary/templates/books/edit.html [hunk 4]
```diff
@@ -82,36 +105,16 @@ <h1>$:title</h1>
...
-            $:render_template("books/author-autocomplete", authors, "authors", "work--authors")
-        </div>
-    </div>
-
+        $ authors = work.authors and [a.author for a in work.authors]
+        <h1 class="editFormBookTitle">$this_title</h1>
+        $if authors:
+            $ all_authors = "By"
+            $for author in authors:
+                $ all_authors = all_authors + " " + author.name + ","
```
Comment (cclauss) [inline openlibrary/templates/books/edit.html:113]:
```suggestion
            $ all_authors = "By {},".format(" ".join(a.author.name for author in authors))
```

File: openlibrary/templates/type/edition/book_title.html [hunk 5]
```diff
@@ -5,11 +5,11 @@
...
-      $if edition.edition_title or edition.edition_name:
-        $if edition.subtitle:
-          &mdash;
-        $edition.edition_title $edition.edition_name
+      $edition.subtitle
+    </h2>
+  $if edition.edition_name:
+    <div class="work-line">
+      $edition.edition_name
+    </div>
```
Comment (cclauss) [inline openlibrary/templates/type/edition/book_title.html:15]: Add \n to the end of the file to get rid of ⛔

File: openlibrary/templates/books/edit.html [hunk 6]
```diff
@@ -149,11 +166,8 @@ <h1>$:title</h1>
 
         $:macros.EditButtons(comment=work.comment_)
 
-        $if ctx.user and (ctx.user.is_admin() or ctx.user.is_librarian()):
-            <div class="adminOnly" style="position:absolute;top:20px;right:20px;"><button type="submit" value="$_('Delete Record')" name="_delete" title="$_('Delete this book?')" id="deleteTop">$_("Delete Record")</button></div>
-
     </form>
-
+z
```
Comment (tabshaikh) [inline openlibrary/templates/books/edit.html:170]:
```suggestion

```

File: openlibrary/templates/books/edit/about.html [hunk 7]
```diff
@@ -67,4 +67,4 @@
         </div>
     </div>
 
-</fieldset>
+</fieldset>
```
Comment (tabshaikh) [inline openlibrary/templates/books/edit/about.html:70]:
```suggestion
</fieldset>

```

File: openlibrary/templates/books/edit/edition.html [hunk 8]
```diff
@@ -354,7 +295,7 @@
                     <input name="edition--by_statement" type="text" id="edition-by_statement" value="$book.by_statement" />
                 </div>
             </div>
-
+
```
Comment (tabshaikh) [inline openlibrary/templates/books/edit/edition.html:298]:
```suggestion
```

File: openlibrary/templates/type/edition/book_title.html [hunk 9]
```diff
@@ -5,11 +5,11 @@
...
```
Comment (tabshaikh) [inline openlibrary/templates/type/edition/book_title.html:15]:
```suggestion
    </div>

```

File: openlibrary/templates/books/author-autocomplete.html [hunk 10]
```diff
@@ -32,8 +32,8 @@
         </div>
 
 $jsdef render_author(name_path, dict_path, disabled, i, author):
-    <div class="input">
-        <input name="$name_path--$i" class="author author-autocomplete" type="text" id="author-$i" value="$author.name" $cond(disabled, 'disabled', '')>
+    <div class="input" style="display: flex;">
+        <input style="flex:1;" name="$name_path--$i" class="author author-autocomplete" type="text" id="author-$i" value="$author.name" $cond(disabled, 'disabled', '')>
```
Comment (mekarpeles) [inline openlibrary/templates/books/author-autocomplete.html:36]: inline classes? Is there a way around this

**Pull request merged**